### DISCUSSION TITLE
Transparent materials not appearing in Scene Viewer

### DISCUSSION THREAD

**Discussion by GabrielArakaki** [google/model-viewer#3637] [Q&A]
### Description

Recently we've faced a few issues in some android devices where transparent materials don't appear at all on scene viewer.
We don't know yet if this is device-related or app-version-related, but we do understand that some Android devices have this issue and others don't.

On our example below we have 4 spheres on our GLB but only the top 2 have transparency.

https://user-images.githubusercontent.com/7118128/125706685-a07ba387-f0c5-408f-abb8-b0b8150f9bb2.mp4

*ps: Everything is working fine on model-viewer's webxr experience. We are switching the experience, only tweaking a few things regarding instructions commands and other usability stuff.

Environment tested:

Model name: Galaxy S10e
Model number: SM-G970F
Android version: 11
Google App: 12.25.10.23.arm64
Google Play Services for AR: 1.25.211460203

#### Live Demo

https://bolder-worried-stetson.glitch.me/

### Version

- model-viewer: latest

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [x] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
@tpsiaki @AdrianAtGoogle Another Scene Viewer issue.

**Comment by AdrianAtGoogle** [comment]
Hi @GabrielArakaki thank you for bringing this to our attention.

The transparent spheres not appearing seems to be chipset related (it does not repro on pixel5); We have heard scattered reports of transparency issues on exynos/mali chipsets but haven't had such an unambiguous example asset to test against.  We will follow up when we know more.
